drivers/hmis/app/graphql/types/hmis_schema/query_type.rb

```diff
@@ -502,8 +502,17 @@ def project_can_accept_referral(destination_project_id:, source_enrollment_id:)
 
     field :client_detail_forms, [Types::HmisSchema::OccurrencePointForm], null: false, description: 'Custom forms for collecting and/or displaying custom details for a Client (outside of the Client demographics form)'
     def client_detail_forms
-      # No authorization required, this just resolving application configuration
-      Hmis::Form::Instance.active.with_role(:CLIENT_DETAIL).published.sort_by_option(:form_title)
+      # No authorization required, this just resolves application configuration
+      Hmis::Form::Instance.active.with_role(:CLIENT_DETAIL).published.sort_by_option(:form_title).
```

Comment:
This is to resolve test failures here: https://github.com/greenriver/hmis-warehouse/actions/runs/11846175768/job/33013264262?pr=4877

This definitely begins to feel like scope creep, so if you prefer, we can leave it as-is and have the `HmisSchema::OccurrencePointForm` provide a default value (as it did before). I think I don't have a strong preference, although I guess my feeling is that it's more graceful for the schema OccurrencePointForm to always be backed by an OpenStruct, as opposed to sometimes an OpenStruct and sometimes an Instance?

Reply:
Oh I forgot about this one. Yeah I agree a graphql type should always be backed by the same object type (Open Struct, now)